homeassistant/components/graphite/__init__.py

```diff
@@ -139,18 +139,21 @@ def run(self):
                 _LOGGER.debug("Event processing thread stopped")
                 self._queue.task_done()
                 return
-            if event.event_type == EVENT_STATE_CHANGED and event.data.get("new_state"):
-                _LOGGER.debug(
-                    "Processing STATE_CHANGED event for %s", event.data["entity_id"]
-                )
-                try:
-                    self._report_attributes(
-                        event.data["entity_id"], event.data["new_state"]
+            if event.event_type == EVENT_STATE_CHANGED:
+                if event.data.get("new_state"):
```

Comment:
Please invert this check and if true log the debug skipping message and continue. Then we can outdent below.

Reply:
Thx, done.